make/data/symbols/jdk.jpackage-E.sym.txt

```diff
@@ -26,6 +26,6 @@
 # ### THIS FILE IS AUTOMATICALLY GENERATED. DO NOT EDIT. ###
 # ##########################################################
 #
-module name jdk.incubator.jpackage
-header requires name\u0020;java.base\u0020;flags\u0020;8000,name\u0020;jdk.jlink\u0020;flags\u0020;0,name\u0020;java.desktop\u0020;flags\u0020;0 uses jdk/incubator/jpackage/internal/Bundler,jdk/incubator/jpackage/internal/Bundlers provides interface\u0020;java/util/spi/ToolProvider\u0020;impls\u0020;jdk/incubator/jpackage/internal/JPackageToolProvider,interface\u0020;jdk/incubator/jpackage/internal/Bundler\u0020;impls\u0020;jdk/incubator/jpackage/internal/LinuxAppBundler\u005C;u002C;jdk/incubator/jpackage/internal/LinuxDebBundler\u005C;u002C;jdk/incubator/jpackage/internal/LinuxRpmBundler,interface\u0020;jdk/incubator/jpackage/internal/Bundlers\u0020;impls\u0020;jdk/incubator/jpackage/internal/BasicBundlers target linux-amd64 resolution 9 flags 8000
+module name jdk.jpackage
```

Comment:
I think you need to revert this. Note this comment:

```
 # ### THIS FILE IS AUTOMATICALLY GENERATED. DO NOT EDIT. ###
```

Reply:
reverted